src/Shared/BrowserTesting/src/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
This file and the PublicAPI.Unshipped.txt file lack `#nullable enable` first lines. _Always_ create these files by copying eng/PublicAPI.empty.txt

Reply:
I've removed this completely for now